api.go

```diff
@@ -0,0 +1,44 @@
+package forwarder
+
+import (
+	"net/http"
+	"net/http/pprof"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// APIHandler returns http.Handler serving API endpoints.
+// It provides health and readiness endpoints prometheus metrics, and pprof debug endpoints.
+func APIHandler(s *HTTPServer, r prometheus.Gatherer) http.Handler {
+	m := http.NewServeMux()
+	m.HandleFunc("/api/v1/health", healthHandler)
+	m.HandleFunc("/api/v1/ready", readyHandler(s))
+
+	m.HandleFunc("/metrics", promhttp.HandlerFor(r, promhttp.HandlerOpts{}).ServeHTTP)
+
+	m.HandleFunc("/debug/pprof/", pprof.Index)
+	m.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
```

Comment:
If we're going to export the cmdline / config, we need to sanitize credentials. I'm fine with making that an issue for later though.

Reply:
I'll remove that line and open an issue to provide sanitized config.